ci/pipeline-production.yml

```diff
@@ -0,0 +1,434 @@
+            CF_APP_NAME: pages-((deploy-env))
+        # - task: restage-queues-ui
+        #   file: src/ci/partials/restage.yml
+        #   image: cf-image
+        #   params:
+        #     <<: *env-cf
+        #     CF_APP_NAME: pages-queues-ui-((deploy-env))
+
+  - name: sites-builds-checks-((deploy-env))
+    plan:
+      - get: src
+        resource: src-((deploy-env))
+        passed: [set-pipeline]
+        params: {depth: 1}
+      - get: cf-image
+      - get: five-minutes
+        trigger: true
+      - in_parallel:
+        - task: queued-builds-check
+          file: src/ci/partials/run-task.yml
+          image: cf-image
+          params:
+            <<: *env-cf
+            CF_APP_NAME: pages-((deploy-env))
+            CF_TASK_NAME: queued-builds-check
+            CF_TASK_COMMAND: yarn queued-builds-check ((support-email))
+        on_failure:
+          in_parallel:
+            - put: slack
+              params:
+                text: |
+                  :x: FAILED: sites builds checks on pages ((deploy-env))
+                  <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME?vars.deploy-env="((deploy-env))"|View build details>
+                channel: ((slack-channel))
+                username: ((slack-username))
+                icon_url: ((slack-icon-url))
+
+  - name: set-pipeline
+    plan:
+    - get: src
+      resource: src-((deploy-env))
+      params: {depth: 1}
+      trigger: true
+    - set_pipeline: core
+      file: src/ci/pipeline.yml
+      instance_vars:
+        deploy-env: ((deploy-env))
+
+  - name: update-release-branch
+    plan:
+    - get: src
+      resource: src-((deploy-env))
+      trigger: true
+    - get: cf-image
+    - task: update-release-branch
+      config:
+        platform: linux
+        inputs:
+          - name: src
+        params:
+          GH_EMAIL: ((pages-operations-github-user-info.email))
+          GH_USERNAME: ((pages-operations-github-user-info.username))
+          GH_BOT_GPG_KEY: ((pages-operations-github-user-gpg.private_key))
+          GH_BOT_SSH_KEY: ((pages-gpg-operations-github-sshkey.private_key))
+          GH_BOT_GPG_TRUST: ((pages-operations-github-user-gpg-trust))
+          GH_TOKEN: ((pages-operations-ci-github-token))
+        run:
+          dir: src
+          path: ci/tasks/update-release-branch.sh
+
+############################
+#  RESOURCES
+
+resources:
+  - name: src-((deploy-env))
+    type: git
+    icon: github
+    source:
+      uri: ((git-base-url))/((pages-repository-path))
+      branch: main
+      commit_verification_keys: ((cloud-gov-pages-gpg-keys))
+      private_key: ((pages-gpg-operations-github-sshkey.private_key))
+
+  - name: src-((deploy-env))-tagged
+    type: git
+    icon: github
+    source:
+      uri: ((git-base-url))/((pages-repository-path))
+      branch: main
+      commit_verification_keys: ((cloud-gov-pages-gpg-keys))
+      tag_filter: 0.*.*
```

Comment:
Can we update the docs on how to pin the resource in Concourse? See https://concourse-ci.org/resource-versions.html#version-pinning